extensions/typedoc.js

```diff
@@ -0,0 +1,40 @@
+Chomp.registerTemplate('typedoc-generator', function (task) {
```

Comment:
V. late to the party, but neat! Not sure why I didn't think of using the task runner to... erm... run tasks 😅